docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Discover insecure SSH access with Teleport Policy
+description: Describes how to enable SSH Keys Scanning using Teleport Policy and Access Graph.
+---
+
+Organizations often struggle to maintain a secure environment due to the lack of visibility into access patterns.
+Long-lived SSH keys, for example, can pose a significant security risk as they can be compromised or mismanaged while
+still providing non-auditable access to critical systems.
+
+With Teleport Policy's Access Graph, you can gain insights on how SSH keys are used within your environment. By scanning
+all SSH authorized keys present on your servers and SSH Private Keys present on company managed laptops, Teleport Policy can
+provide a visual representation of the access patterns and help you enhance the security of your environment.
+
+This functionality gives you insights into the following areas:
+
+- Which servers have SSH Authorized Keys that could be used to bypass Teleport?
+- Which users have SSH Private Keys that grant access to SSH servers?
+- Which laptops have unprotected SSH Private Keys?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise edition customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
+
+## How it works
+
+Teleport and Teleport Policy's Access Graph synchronize various resources, including SSH authorized keys and private keys.
+
+These resources are then visualized using the graph representation detailed in the
+[Access Graph page](../teleport-policy/getting-started-policy.mdx).
+
+### Importing an SSH Node's Authorized Keys
+
+Teleport Cluster Nodes running the Teleport SSH Service continuously scan for SSH Authorized Keys present on the server.
```

Comment:
This is not directly in scope of this doc but reading this I got a thought that technically I think it would make sense for any kind of agent to report authorized keys, not just SSH nodes. For example, you could have a server that runs database agent that may also be available via direct SSH access.

Reply:
the logic behind my decision is that in that case, you don't have any other way of accessing the node besides using ssh firectly. all patterns will always be insecure since they aren't auditable. 

We can change it and make all services reporting